view/src/components/produce.js

```diff
@@ -0,0 +1,723 @@
+                                  {produce.shippingPresetTemperature}
+                                </Typography>
+                              </Box>
+                              <Box p={3}>
+                                <Typography
+                                  className={classes.pos}
+                                  color="textSecondary"
+                                >
+                                  Pricing (in USD / lb):
+                                </Typography>
+                                <Typography variant="body2" component="p">
+                                  USDA Price: ${produce.price}
+                                  <br />
+                                  Average Price Paid by Farmlink: $
+                                  {produce.pricePaid}
+                                </Typography>
+                              </Box>
+                              <Box p={3}>
+                                <Typography
+                                  className={classes.pos}
+                                  color="textSecondary"
+                                >
+                                  Internal Statistics:
+                                </Typography>
+                                <Typography variant="body2" component="p">
+                                  Amount Moved by FarmLink (lbs):{" "}
+                                  {produce.amountMoved}
+                                </Typography>
+                              </Box>
+                            </Box>
+                          </CardContent>
+                          <CardActions>
+                            <Button
+                              size="small"
+                              color="primary"
+                              onClick={() => this.handleViewOpen({ produce })}
+                            >
+                              {" "}
+                              View{" "}
+                            </Button>
+                            <Button
+                              size="small"
+                              color="primary"
+                              onClick={() =>
+                                this.handleEditClickOpen({ produce })
+                              }
+                            >
+                              Edit
+                            </Button>
+                            <Button
+                              size="small"
+                              color="primary"
+                              onClick={() =>
+                                this.deleteTodoHandler({ produce })
+                              }
+                            >
+                              Delete
+                            </Button>
+                          </CardActions>
+                        </Card>
+                      </Grid>
+                    ))}
+                  </div>
+                )}
+              />
```

Comment:
I think that you can make the code more readable by naming the event handler.  According to the [documentation](https://reactjs.org/docs/faq-functions.html), you can bind a function to a component instance.

Reply:
(@andrewhojel )

Reply:
(Will discuss during standup today -- dif views across the team)

Reply:
I think this is related to the the `renderResults` of `SearchResults`

Reply:
resolved this in PR #37